app/src/main/res/values/strings.xml

```diff
@@ -66,6 +65,7 @@
   <string name="patiententry_error_dateofbirth_is_in_future">Fix patient’s date of birth</string>
   <string name="patiententry_error_phonenumber_length_less">Phone number cannot be less than 6 digits</string>
   <string name="patiententry_error_phonenumber_length_more">Phone number cannot be more than 12 digits</string>
+  <string name="patiententry_identifier">BP Passports</string>
```

Comment:
Shouldn't we be using `plurals` for this string?

Reply:
Good point. Not that I think about it, when we are registering a patient, there will never be more than one passport. This should be a singular instead. I'll confirm with Akshay.

Reply:
Confirmed with Akshay offline, this will be a singular string.